packages/dcos-integration-test/extra/test_endpoints.py

```diff
@@ -141,3 +141,39 @@ def test_if_we_have_capabilities(cluster):
     )
     assert r.status_code == 200
     assert {'name': 'PACKAGE_MANAGEMENT'} in r.json()['capabilities']
+
+
+def test_if_overlay_master_is_up(cluster):
+    r = cluster.get('/mesos/overlay-master/state')
+
+    assert r.status_code == 200
```

Comment:
`assert r.status_code == 200` ends up producing really hard to debug / track down errors when it fails (you don't get the status code or response body printed which are both usually critical in solving). Please use `r.raise_for_status()` or do a `assert r.ok, "status_code: {}, content: {}".format(r.status_code, r.content))